crates/sui-types/src/messages.rs

```diff
@@ -745,7 +728,10 @@ impl SignedTransaction {
             transaction_digest: OnceCell::new(),
             is_verified: false,
             data,
-            tx_signature: Signature::new_empty(),
+            // Arbitrary keypair
+            tx_signature: Secp256k1SuiSignature::from_bytes(&[0; Secp256k1SuiSignature::LENGTH])
```

Comment:
It doesn't matter which kind of signature we put here - it won't be verified anyways.